src/CMakeLists.txt

```diff
@@ -14,16 +14,22 @@ if (CMAKE_CXX_COMPILER_ID MATCHES "GNU|Clang")
 	add_link_options ("-fvisibility=hidden")
 endif ()
 
+if (LOVR)
+  set(LUA_LIBRARIES ${LOVR_LUA})
+else()
 ### Dependencies
-find_package (LuaJIT)
-if (LUAJIT_FOUND)
-	set(LUA_INCLUDE_DIR ${LUAJIT_INCLUDE_DIR})
-	set(LUA_LIBRARIES ${LUAJIT_LIBRARY})
-else ()
-	find_package (Lua 5.1 REQUIRED)
+	find_package (LuaJIT)
+	if (LUAJIT_FOUND)
+		set(LUA_INCLUDE_DIR ${LUAJIT_INCLUDE_DIR})
+		set(LUA_LIBRARIES ${LUAJIT_LIBRARY})
+	else ()
+		find_package (Lua 5.1 REQUIRED)
+	endif ()
+
+	include_directories (${LUA_INCLUDE_DIR})
 endif ()
 
-include_directories (${LUA_INCLUDE_DIR})
+
```

Comment:
This will only call `include_directories` when Lua 5.1 is used instead of LuaJIT.

Reply:
Isn't this identical to how the code worked before?  Just wrapped inside a check for the LOVR flag

Reply:
Oh sorry I misread the code. Yeah looks good.